### ISSUE TITLE
edge colouring

### ISSUE THREAD

**Issue by jjkoehorst** [Edinburgh-Genome-Foundry/DnaFeaturesViewer#14]
Very nice package and works also very smooth with larger datasets but the genes since they are so small are becoming a single line. 
To visualise different categories I would like to apply colouring to them. However is it also possible to colour the edges? Since the only thing I can see are the edges since they are so small and all turn into a black line instead of any other colour.

**Comment by Zulko** [comment]
Thanks for the feedback !

I have now added a ``linecolor`` parameter to the GraphicFeatures class (before that, it was hardcoded to "black"). I updated both Github and PyPI. It should do what you want.

However, in your particular case, the best solution is to set the ``linewidth`` property (already in your current version) to 0.

I don't know how you create your GraphicRecords, but just doing that a-posteriori works:

```
for feature in my_graphic_record.features:
    feature.linewidth = 0
```
